extensions/wikia/AdEngine/js/AdConfig2Late.js

```diff
@@ -81,6 +83,14 @@ define('ext.wikia.adEngine.adConfigLate', [
 			}
 		}
 
+		if (context.providers.monetizationService) {
+			if (instantGlobals.wgSitewideDisableMonetizationService) {
+				log('MonetizationService disabled by DR. No ads', 'warn', logGroup);
+				return [];
+			}
+			return [adProviderMonetizationService];
```

Comment:
This makes Monetization Service the only ad provider for the late queue, breaking Liftium and Evolve.